Readability.js

```diff
@@ -2263,6 +2268,7 @@ Readability.prototype = {
       title: this._articleTitle,
       byline: metadata.byline || this._articleByline,
       dir: this._articleDir,
+      lang: this._articleLang,
```

Comment:
Could we make sure we have a test for this? These are the steps you should take for this:

1. Assert that Readability.parse() extracts the lang attribute. You should add this after this line https://github.com/mozilla/readability/blob/master/test/test-readability.js#L178. 
2. Generate a test case where the original document has a "lang" attribute on the root. You can generate this test case using the following:

```
node test/generate-testcase.js nytimes-5 https://www.nytimes.com/es/
```

Once you run that command you can manually check the expected-metadata.json file under the generated directory. If you used the above example then the "lang" property will have an "es" value.

3. Make sure to run `npm test && npm run lint`